db/pre-re/status.yml

```diff
@@ -115,6 +115,13 @@ Body:
       Inspiration: true
       Warmer: true
       Gvg_Freez: true
+      Stone: true
```

Comment:
Need to consider what Lemongrass said, that we could do something like "FailAllOpt1Except:" instead. (Or we do that later.)